### ISSUE TITLE
Create a shipment - submitting the draft

### ISSUE THREAD

**Issue by Francois-Jo** [trexitycode/openapi#13]
Hi team, I am using the Trexity API with sandbox environment and I am wondering if there is any way of submitting the draft of the created route.
For now I am only able to draft a route and update the parcels. I went through the API documentation but I was not able to figure out if there is a way of submiting a drafted route. (I guess it might not be possible hence when we submit manually, we have to agree on some conditions...).
Best regards,
Francois

**Comment by dschnare** [comment]
Hey @Francois-Jo, you can schedule the shipments when creating them or you can schedule the shipment with a request to update the shipment. However, in the sandbox environment there are no couriers so beyond scheduling, the shipment lifecycle stops there.

**Comment by Francois-Jo** [comment]
Hi @dschnare , thank you for your quick reply. Scheduling the shipment is fine, it is more that when I create a shipment, it stays in the status "draft" :
![image](https://github.com/user-attachments/assets/5576f21d-d5ec-43ff-a38a-d66dedb64c66)

and there is no way to "submit" it : we can only confirm delivery manually
![image](https://github.com/user-attachments/assets/b7e6e0bb-1fd9-4850-8281-dd566d78dfea)

Is there a way to "confirm the delivery" through the API ?

**Comment by dschnare** [comment]
That's correct. You can schedule the delivery via the UI (your second screenshot) OR via the API using the "pickupStartAt" field in the request to create the shipment or update the shipment. However, the lifecycle of the shipment will not proceed past this point.
